packages/terra-tag/src/Tag.scss

```diff
@@ -0,0 +1,72 @@
+
+:local {
+  .tag {
+    border-radius: 0.2142rem;
+    border-style: var(--terra-tag-border-style, solid);
+    border-width: 0.07142rem;
+    cursor: pointer;
+    display: inline-flex;
+    font-family: inherit;
+    font-size: var(--terra-tag-font-size, 1rem);
+    font-weight: var(--terra-tag-font-weight, normal);
+    line-height: var(--terra-tag-line-height, 1.14285rem);
+    margin-bottom: var(--terra-tag-margin-bottom, 0.357rem);
```

Comment:
Why aren't we using the shorthand for margin and padding?

Reply:
Because on my previous [PR](https://github.com/cerner/terra-core/pull/1222) I was asked to split it up.